lib/content_importer.rb

```diff
@@ -0,0 +1,45 @@
+module ContentImporter
+  def self.import_from_hash(set_class, set_element_class, set_class_attributes, options = {})
+    status = {:class => name, :description => set_class_attributes["description"], :children => []}
+    profiles = set_class_attributes.dup.delete(set_element_class.name) do |_k|
```

Comment:
@chessbyte The `.dup` here is causing the test failure because the key representing the `set_element_class` (MiqAlert, for example) needs to be removed from the attrs before creating the new instance. The original code was doing this.